package.json

```diff
@@ -8,7 +8,7 @@
   },
   "scripts": {
     "build": "lerna run build && shx cp -r ./src/packages/ganache/dist/web ./docs/assets/js/ganache",
-    "clean": "npx shx rm -rf node_modules && npx lerna clean -y && npm run tsc.clean",
+    "clean": "npm run tsc.clean && npx lerna clean -y && npx shx rm -rf node_modules",
```

Comment:
I had to reverse these because the first command `npx shx rm -rf node_modules` removes a non-global installation of lerna, causing the following commands to fail

Reply:
lerna is run with `npx` here, so that shouldn't be a problem. There might be another issue with the way ganache is configured. hmmm

Reply:
All I know is running `npm run clean` would finally get to `npm run tsc.clean` and I would get `lerna not found`. I'm guessing this has been working for you because you have a globally installed version of lerna, which isn't a requirement of this repo.

After changing the order of this command, I can run `npm run clean` and choosing to not look into the issue further and call it done. I don't think this is a configuration issue, nor do I think I should have to reinstall lerna (because that's what npx will do if I don't have it) just to finish running the clean command lol

Reply:
I'm good with the change; it make sense... I just found it strange.


Only commenting this just in case we need to come back to this one day:

I don't have lerna installed globally, and `npm run clean` (`npx shx rm -rf node_modules && npx lerna clean -y && npm run tsc.clean`) looks to be working on my machine.


```console
david at david-kubuntu in ~/work/ganache-core on chore/separate-options-from-flavors-next [!$]
$ lerna
lerna: command not found

david at david-kubuntu in ~/work/ganache-core on chore/separate-options-from-flavors-next [!$]
$ npx shx rm -rf node_modules

david at david-kubuntu in ~/work/ganache-core on chore/separate-options-from-flavors-next [!$]
$ npm run clean

> root@ clean /home/david/work/ganache-core
> npx shx rm -rf node_modules && npx lerna clean -y && npm run tsc.clean

npx: installed 21 in 3.73s
npx: installed 766 in 34.004s
lerna notice cli v3.22.1
lerna info versioning independent
lerna info clean removing /home/david/work/ganache-core/src/chains/ethereum/ethereum/node_modules
lerna info clean removing /home/david/work/ganache-core/src/chains/ethereum/options/node_modules
lerna info clean removing /home/david/work/ganache-core/src/chains/ethereum/utils/node_modules
lerna info clean removing /home/david/work/ganache-core/src/chains/tezos/options/node_modules
lerna info clean removing /home/david/work/ganache-core/src/chains/tezos/tezos/node_modules
lerna info clean removing /home/david/work/ganache-core/src/packages/cli/node_modules
lerna info clean removing /home/david/work/ganache-core/src/packages/core/node_modules
lerna info clean removing /home/david/work/ganache-core/src/packages/flavors/node_modules
lerna info clean removing /home/david/work/ganache-core/src/packages/ganache/node_modules
lerna info clean removing /home/david/work/ganache-core/src/packages/options/node_modules
lerna info clean removing /home/david/work/ganache-core/src/packages/promise-queue/node_modules
lerna info clean removing /home/david/work/ganache-core/src/packages/utils/node_modules
lerna success clean finished

> root@ tsc.clean /home/david/work/ganache-core
> npx lerna exec -- npx shx rm -rf lib dist

npx: installed 766 in 31.316s
lerna notice cli v3.22.1
lerna info versioning independent
lerna info Executing command in 12 packages: "npx shx rm -rf lib dist"
npx: installed 21 in 3.513sact:shx: verb lock using /home/david/.npm/_locks/staging-0393b40e8a8e169d.lock for /home/david/.npm/_npx/936938/lib/node_modules/.staging
npx: installed 21 in 3.599sesh-package-json:shx: timing action:finalize Completed in 21ms
npx: installed 21 in 3.643s
npx: installed 21 in 3.54s
npx: installed 21 in 3.545s
npx: installed 21 in 3.464s
npx: installed 21 in 3.585s
npx: installed 21 in 3.268s
npx: installed 21 in 3.468s
npx: installed 21 in 3.529s
npx: installed 21 in 3.39s
npx: installed 21 in 3.337s
lerna success exec Executed command in 12 packages: "npx shx rm -rf lib dist"
```